specification/communication/data-plane/CallAutomation/preview/2024-09-01-preview/examples/CallMedia_InterruptAudioAndAnnounce.json

```diff
@@ -0,0 +1,28 @@
+{
+  "parameters": {
+    "endpoint": "https://contoso.communications.azure.com",
```

Comment:
We've had previous reviews with Franko and gave feedback to try adding a new property to the existing Play API instead of creating a slightly different copy of the route. What's the state of this API?

Reply:
This is actually not part of my change but without it some build failed so I added the sample based on the swagger